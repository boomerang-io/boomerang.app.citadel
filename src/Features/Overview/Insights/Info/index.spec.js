import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import { Info } from "./index";

const info = {
  type: "test",
  title: "Test title",
  content: "Test content text",
  count: "---"
};

describe("Info --- Snapshot", () => {
  it("Capturing Snapshot of Info", () => {
    const renderedValue = renderer
      .create(
        <Info
          info={info}
        />
      )
      .toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});

describe("Info --- Shallow render", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Info
        info={info}
      />
    );
  });

  it("Render the DUMB component", () => {
    expect(wrapper.length).toEqual(1);
  });
});
