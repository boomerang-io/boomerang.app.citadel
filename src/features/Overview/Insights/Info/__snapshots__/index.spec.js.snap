// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info --- Snapshot Capturing Snapshot of Info 1`] = `
<div
  className="bx--tile"
  data-testid="tile-info"
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "height": "10rem",
      "padding": "0",
      "width": "100%",
    }
  }
>
  <div
    className="counter"
    style={
      Object {
        "color": "#ffaa9d",
      }
    }
  >
    ---
  </div>
  <div
    className="divider"
  />
  <div
    className="text"
  >
    <h3
      className="title"
    >
      Test title
    </h3>
    <p
      className="content"
    >
      Test content text
    </p>
  </div>
</div>
`;
